README.md

```diff
@@ -1,45 +1,63 @@
-[PHP](https://github.com/mongodb/mongo-php-driver) and
-[HHVM](https://github.com/mongodb/mongo-hhvm-driver) (i.e. the `mongodb`
-extension).
+[![Build Status](https://api.travis-ci.org/mongodb/mongo-php-library.png?branch=master)](https://travis-ci.org/mongodb/mongo-php-library)
+
+This library provides a high-level abstraction around the lower-level
+[PHP driver](https://github.com/mongodb/mongo-php-driver) (`mongodb` extension).
 
 While the extension provides a limited API for executing commands, queries, and
 write operations, this library implements an API similar to that of the
-[legacy PHP driver](http://php.net/manual/en/book.mongo.php). It contains
+[legacy PHP driver](https://php.net/manual/en/book.mongo.php). It contains
 abstractions for client, database, and collection objects, and provides methods
 for CRUD operations and common commands (e.g. index and collection management).
 
 If you are developing an application with MongoDB, you should consider using
 this library, or another high-level abstraction, instead of the extension alone.
 
-For further information about the architecture of this library and the `mongodb`
-extension, see:
-
- - http://www.mongodb.com/blog/post/call-feedback-new-php-and-hhvm-drivers
+Additional information about the architecture of this library and the `mongodb`
+extension may be found in
+[Architecture Overview](https://php.net/manual/en/mongodb.overview.php).
 
 ## Documentation
 
  - https://docs.mongodb.com/php-library/
+ - https://docs.mongodb.com/ecosystem/drivers/php/
 
-# Installation
-
-As a high-level abstraction for the driver, this library naturally requires that
-the [`mongodb` extension be installed](http://php.net/manual/en/mongodb.installation.php):
-
-    $ pecl install mongodb
-    $ echo "extension=mongodb.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`
+## Installation
 
 The preferred method of installing this library is with
 [Composer](https://getcomposer.org/) by running the following from your project
 root:
 
     $ composer require mongodb/mongodb
 
+Additional installation instructions may be found in the
+[library documentation](https://docs.mongodb.com/php-library/current/tutorial/install-php-library/).
+
+Since this library is a high-level abstraction for the driver, it also requires
+that the `mongodb` extension be installed:
+
+    $ pecl install mongodb
+    $ echo "extension=mongodb.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`
+
+Additional installation instructions for the extension may be found in its
+[PHP.net documentation](https://php.net/manual/en/mongodb.installation.php).
+
 ## Reporting Issues
 
-Please use the following form to report any issues:
+Issues may be reported in either
+[GitHub](https://github.com/mongodb/mongo-php-library/issues/new) or
+[JIRA](https://jira.mongodb.org/secure/CreateIssue!default.jspa?project-field=PHPLIB).
```

Comment:
@derickr I assume you'd like this changed as well (per https://github.com/mongodb/mongo-php-driver/pull/769#discussion_r170856962)?